src/pluto/pluto.py

```diff
@@ -20,12 +20,13 @@ def pluto():
 
     branches = get_branches()
     branch = st.sidebar.selectbox(
-        "select a branch",
-        branches,
-        index=branches.index("main"),
+        "select a branch", branches, index=branches.index("main"),
```

Comment:
Strange, when I run the black formatting it formats this chunk as 
```python
    branch = st.sidebar.selectbox(
        "select a branch",
        branches,
        index=branches.index("main"),
    )
```

Reply:
not quite sure what causes the variation, is this one the only difference?

Reply:
this change happens 3 times where it splits arguments across multiple lines. could it have to do with the version of black we are using?

Reply:
yes, I guess it is due to the version. I update black to the latest version, now it seems look the same